### ISSUE TITLE
linux_latest broken on aarch64

### ISSUE THREAD

**Issue by lheckemann** [NixOS/nixpkgs#75650]
**Describe the bug**
Fails to build with some not-supported instructions:
<details>

```/build/ccs2D7Jj.s: Assembler messages:
/build/ccs2D7Jj.s:50: Error: selected processor does not support `aese v7.16b,v1.16b'
/build/ccs2D7Jj.s:51: Error: selected processor does not support `aesmc v7.16b,v7.16b'
/build/ccs2D7Jj.s:54: Error: selected processor does not support `aese v8.16b,v1.16b'
/build/ccs2D7Jj.s:55: Error: selected processor does not support `aesmc v8.16b,v8.16b'
/build/ccs2D7Jj.s:62: Error: selected processor does not support `aese v15.16b,v1.16b'
/build/ccs2D7Jj.s:63: Error: selected processor does not support `aesmc v15.16b,v15.16b'
/build/ccs2D7Jj.s:66: Error: selected processor does not support `aese v14.16b,v1.16b'
/build/ccs2D7Jj.s:67: Error: selected processor does not support `aesmc v14.16b,v14.16b'
/build/ccs2D7Jj.s:77: Error: selected processor does not support `aese v10.16b,v1.16b'
/build/ccs2D7Jj.s:78: Error: selected processor does not support `aesmc v10.16b,v10.16b'
/build/ccs2D7Jj.s:447: Error: selected processor does not support `aese v8.16b,v3.16b'
/build/ccs2D7Jj.s:448: Error: selected processor does not support `aesmc v8.16b,v8.16b'
/build/ccs2D7Jj.s:451: Error: selected processor does not support `aese v9.16b,v3.16b'
/build/ccs2D7Jj.s:452: Error: selected processor does not support `aesmc v9.16b,v9.16b'
/build/ccs2D7Jj.s:460: Error: selected processor does not support `aese v8.16b,v3.16b'
/build/ccs2D7Jj.s:461: Error: selected processor does not support `aesmc v8.16b,v8.16b'
/build/ccs2D7Jj.s:467: Error: selected processor does not support `aese v10.16b,v3.16b'
/build/ccs2D7Jj.s:468: Error: selected processor does not support `aesmc v10.16b,v10.16b'
/build/ccs2D7Jj.s:475: Error: selected processor does not support `aese v10.16b,v3.16b'
/build/ccs2D7Jj.s:476: Error: selected processor does not support `aesmc v10.16b,v10.16b'
/build/ccs2D7Jj.s:835: Error: selected processor does not support `aese v1.16b,v4.16b'
/build/ccs2D7Jj.s:836: Error: selected processor does not support `aesmc v1.16b,v1.16b'
/build/ccs2D7Jj.s:839: Error: selected processor does not support `aese v0.16b,v4.16b'
/build/ccs2D7Jj.s:840: Error: selected processor does not support `aesmc v0.16b,v0.16b'
/build/ccs2D7Jj.s:848: Error: selected processor does not support `aese v0.16b,v4.16b'
/build/ccs2D7Jj.s:849: Error: selected processor does not support `aesmc v0.16b,v0.16b'
/build/ccs2D7Jj.s:856: Error: selected processor does not support `aese v0.16b,v4.16b'
/build/ccs2D7Jj.s:857: Error: selected processor does not support `aesmc v0.16b,v0.16b'
/build/ccs2D7Jj.s:864: Error: selected processor does not support `aese v9.16b,v4.16b'
/build/ccs2D7Jj.s:865: Error: selected processor does not support `aesmc v9.16b,v9.16b'
/build/ccs2D7Jj.s:1252: Error: selected processor does not support `aese v8.16b,v4.16b'
/build/ccs2D7Jj.s:1253: Error: selected processor does not support `aesmc v8.16b,v8.16b'
/build/ccs2D7Jj.s:1256: Error: selected processor does not support `aese v1.16b,v4.16b'
/build/ccs2D7Jj.s:1257: Error: selected processor does not support `aesmc v1.16b,v1.16b'
/build/ccs2D7Jj.s:1263: Error: selected processor does not support `aese v9.16b,v4.16b'
/build/ccs2D7Jj.s:1264: Error: selected processor does not support `aesmc v9.16b,v9.16b'
/build/ccs2D7Jj.s:1271: Error: selected processor does not support `aese v8.16b,v4.16b'
/build/ccs2D7Jj.s:1272: Error: selected processor does not support `aesmc v8.16b,v8.16b'
/build/ccs2D7Jj.s:1280: Error: selected processor does not support `aese v12.16b,v4.16b'
/build/ccs2D7Jj.s:1281: Error: selected processor does not support `aesmc v12.16b,v12.16b'
/build/ccs2D7Jj.s:1629: Error: selected processor does not support `aese v0.16b,v4.16b'
/build/ccs2D7Jj.s:1630: Error: selected processor does not support `aesmc v0.16b,v0.16b'
/build/ccs2D7Jj.s:1637: Error: selected processor does not support `aese v0.16b,v4.16b'
/build/ccs2D7Jj.s:1638: Error: selected processor does not support `aesmc v0.16b,v0.16b'
/build/ccs2D7Jj.s:1645: Error: selected processor does not support `aese v0.16b,v4.16b'
/build/ccs2D7Jj.s:1646: Error: selected processor does not support `aesmc v0.16b,v0.16b'
/build/ccs2D7Jj.s:1653: Error: selected processor does not support `aese v0.16b,v4.16b'
/build/ccs2D7Jj.s:1654: Error: selected processor does not support `aesmc v0.16b,v0.16b'
/build/ccs2D7Jj.s:1662: Error: selected processor does not support `aese v0.16b,v4.16b'
/build/ccs2D7Jj.s:1663: Error: selected processor does not support `aesmc v0.16b,v0.16b'
make[2]: *** [../scripts/Makefile.build:265: crypto/aegis128-neon-inner.o] Error 1
make[1]: *** [/build/linux-5.4.3/Makefile:1652: crypto] Error 2
make: *** [../Makefile:179: sub-make] Error 2
```
</details>

**To Reproduce**
1. (on an aarch64 machine) `nix-build channel:nixos-19.09 -A linux_latest`

**Expected behavior**
Kernel builds

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute:
# a list of nixos modules affected by the problem
module:
```
May be caused by 46249289d6e11bba8b313930ca2f8064693e6842

**Comment by lheckemann** [comment]
Fixed already by bumping to 5.4.x, 19.09 channel just hasn't caught up yet.

**Comment by lheckemann** [comment]
Never mind, I'm not very good at interpreting bisects.

**Comment by jakobrs** [comment]
I had the same issue (I think?) when trying to install NixOS 19.09 on Raspberry Pi 3B+

**Comment by lheckemann** [comment]
Fixed in fbe321e.
